cerbero/commands/build.py

```diff
@@ -92,9 +94,14 @@ def runargs(self, config, recipes, missing_files=False, force=False,
             ordered_recipes = recipes
         else:
             ordered_recipes = cookbook.list_recipes_deps(recipes)
+        if fetch:
+          self._fetch(config, ordered_recipes, no_deps=False, reset_rdeps=True)
         oven.start_cooking(ordered_recipes, use_binaries, upload_binaries,
                 build_missing)
 
+    def _fetch(self, config, recipes, no_deps=False, reset_rdeps=False):
```

Comment:
no_deps and reset_rdeps are not needed as it's always used with the same values.

Reply:
Yes I prefered it moludar

Reply:
> We should try to upstream this first

Ok

Reply:
yes, we are already upstreaming things, I don't understand the question :) GitLab's account and fluendo's email in the commits.